### PR TITLE
Two more hover styles for Menu

### PR DESCRIPTION
**Pull request by wlatanowicz**

### CODE REVIEW

File: src/lib/Menu.js [hunk 1]
```diff
@@ -109,17 +125,25 @@ const Menu = (props) => (
...
+              <MenuNavLink
+                href={item.link}
+                hoverLine={props.hoverLine}
+                hasSubMenu={true}
+              >
                 {item.name}
-              </NavLink>
-              <Submenu links={item.submenu} />
+              </MenuNavLink>
+              <Submenu links={item.submenu}  hoverLine={props.hoverLine}/>
```
Comment (feimosi) [inline src/lib/Menu.js]: Two spaces before `hoverLine`

File: src/lib/Menu.js [hunk 2]
```diff
@@ -86,19 +89,32 @@ const SubNavItem = styled.li`
...
   <NavMenu>
     { props.links.map((item, index) =>
       <SubNavItem key={item.name + index}>
-        <NavLink href={item.link}>
+        <MenuNavLink
+          href={item.link}
+          hoverLine={props.hoverLine}
+          hasSubMenu={false}
+          isSubMenu={true}
+        >
```
Comment (feimosi) [inline src/lib/Menu.js]: We probably can omit `hasSubMenu` and leave `isSubMenu` as a boolean attributes
```
  isSubMenu
>
```

File: src/lib/Menu.js [hunk 3]
```diff
@@ -109,17 +125,25 @@ const Menu = (props) => (
     <NavMenu>
       {props.links.map((item, index) =>
         item.submenu ? (
-            <SubbedNavItem key={item.name + index}>
-              <NavLink href={item.link}>
+            <SubbedNavItem
+              key={item.name + index}
+              hoverLine={props.hoverLine}
+              hasSubMenu={true}
```
Comment (feimosi) [inline src/lib/Menu.js]: I don't see `SubbedNavItem` using `hasSubMenu`

Comment (wlatanowicz) [reply src/lib/Menu.js]: `SubbedNavItem` extends `NavItem` which uses

File: src/lib/Menu.js (hunk 2 above)
Comment (wlatanowicz) [reply src/lib/Menu.js]: OK

File: src/lib/Menu.js (hunk 1 above)
Comment (wlatanowicz) [reply src/lib/Menu.js]: Yep

File: src/lib/Menu.js (hunk 3 above)
Comment (feimosi) [reply src/lib/Menu.js]: Ok, didn't notice it :+1:

**Pull request merged**